### ISSUE TITLE
Working half way

### ISSUE THREAD

**Issue by Xeohn** [link07/Dark-Crunchyroll-Plugin-Chrome#3]
I would just like to get this issue fixed. What happens is that the page turn mostly white and it is kind of annoying. Here is a picture.
http://i.imgur.com/hBOwOIs.jpg

I hope you would be kind enough to fix it.
sincerely from me.

**Comment by Xeohn** [comment]
Looks like it was another plugin. Never mind.

**Comment by Xeohn** [comment]
Well it looks like the ads that is supposed to be on the site is making the issue. I can't block the ads with out getting my crunchyroll mostly white.

**Comment by link07** [comment]
I've had another report of a similar issue, which ad block do you use? with ublock I don't have an issue, but I could see if I could fix it anyway.

**Comment by Xeohn** [comment]
Well I use one that is called just adblock and another that is called privacybadger. But i think any adblock that removes the big crunchyroll's own ad that is supposed to be at the white area.... I just tried again and now the background that is white area is there with all adblockers dissabled. Also it works to turn your plugin off and on again. Then everything gets dark again.

**Comment by link07** [comment]
I uploaded a new version to the chrome store (version 1.2.1), that should hopefully fix the issue.  I couldn't recreate the exact issue, but I have a feeling I know what is happening.  Please let me know if it didn't fix it.

**Comment by Xeohn** [comment]
No, it does not work, saldy.
http://i.imgur.com/nnfZDLS.png

But I still think it's strage that it works if I turn it manually off and on again. Like it just want to fully load if you tell the plugin spesific to do so.

**Comment by link07** [comment]
Just now noticed that you replied to this.  If you could save the entire webpage (control+s, save as type "Webpage, Complete," and upload that somewhere, I could try to see if I can pinpoint the tag that's being changed, because my next guess is that Adblock is changing a name of a tag somewhere.

**Comment by Xeohn** [comment]
If you coud give me a email address or someting, I can share the folder with you?

**Comment by link07** [comment]
link33985@gmail.com

**Comment by Xeohn** [comment]
Shared it now.

**Comment by link07** [comment]
Tried to fix it again, I think something in the ad block is causing crunchyroll's css to be reloaded, which then gives it higher priority then the addons (whereas chrome normally gives addons css priority), so I had to add some important markers.
Which means I'll probably have to add some more as other spots of missing theme are found.

Let me know if this doesn't fix it or if you see missing theme elsewhere, as I'll have to go through and add important markers everywhere that needs it, but I'm not sure where that is until people tell me.
